app/models/user.rb

```diff
@@ -90,4 +90,10 @@ def self.non_organizer
   def self.without_questionnaire
     non_organizer.left_outer_joins(:questionnaire).where(questionnaires: { id: nil })
   end
+
+  def as_json(options = {})
```

Comment:
This seems a bit strange out of context (adding param to JSON response but not to the model) -- maybe add a comment explaining the reasoning we discussed in Slack?